From 2ec1d168468ba293855836bfa2a7ae8d337d0cb0 Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Tue, 13 Nov 2012 17:42:07 -0500
Subject: [PATCH 3/4] Do SSL CA verification and hostname validation.

---
 ipa-client/ipa-join.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ipa-client/ipa-join.c b/ipa-client/ipa-join.c
index 4c3711319f7ad8e65c366487884731039da5139c..6a9770fe12325202444e002d909e34189697ad95 100644
--- a/ipa-client/ipa-join.c
+++ b/ipa-client/ipa-join.c
@@ -166,8 +166,8 @@ callRPC(char * user_agent,
     memset(curlXportParmsP, 0, sizeof(*curlXportParmsP));
 
     /* Have curl do SSL certificate validation */
-    curlXportParmsP->no_ssl_verifypeer = 1;
-    curlXportParmsP->no_ssl_verifyhost = 1;
+    curlXportParmsP->no_ssl_verifypeer = 0;
+    curlXportParmsP->no_ssl_verifyhost = 0;
     curlXportParmsP->cainfo = "/etc/ipa/ca.crt";
     curlXportParmsP->user_agent = user_agent;
     /* Enable GSSAPI credentials delegation */
-- 
1.7.1

