From 49311df28fad07cdbd986b7ba1d6e3127b8f5776 Mon Sep 17 00:00:00 2001
From: Simo Sorce <simo@redhat.com>
Date: Fri, 26 Jun 2015 20:47:57 -0400
Subject: [PATCH] Simplify adding options in ipachangeconf

Signed-off-by: Simo Sorce <simo@redhat.com>
Reviewed-By: Jan Cholasta <jcholast@redhat.com>
---
 ipa-client/ipaclient/ipachangeconf.py | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/ipa-client/ipaclient/ipachangeconf.py b/ipa-client/ipaclient/ipachangeconf.py
index 42f6496e09404ab2d995f4ec91646f5c710ce723..46132212112ff198e742f530860df4b15ea3312b 100644
--- a/ipa-client/ipaclient/ipachangeconf.py
+++ b/ipa-client/ipaclient/ipachangeconf.py
@@ -526,3 +526,24 @@ class IPAChangeConf:
             except IOError:
                 pass
         return True
+
+    @staticmethod
+    def setOption(name, value):
+        return {'name': name,
+                'type': 'option',
+                'action': 'set',
+                'value': value}
+
+    @staticmethod
+    def rmOption(name):
+        return {'name': name,
+                'type': 'option',
+                'action': 'remove',
+                'value': None}
+
+    @staticmethod
+    def setSection(name, options):
+        return {'name': name,
+                'type': 'section',
+                'action': 'set',
+                'value': options}
-- 
2.6.4

