From 0f5bdfbb3a317bff9b34f85d8eba11e5b999d3f6 Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Thu, 30 Aug 2012 10:59:35 -0400
Subject: [PATCH] Don't check ipa_memcached server status

---
 ipalib/session.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/ipalib/session.py b/ipalib/session.py
index 68b9b264be10e389a0f55758b1920d4174320c0e..3aeb45784f861ad91a946e16999d9305d47e85aa 100644
--- a/ipalib/session.py
+++ b/ipalib/session.py
@@ -849,8 +849,10 @@ class MemcacheSessionManager(SessionManager):
 
         if len(self.servers) == 0:
             return False
-        stats = self.get_server_statistics()
-        return len(self.servers) == len(stats)
+
+        # Work around bug in python-memcached where we can't check on
+        # server status
+        return True
 
     def new_session_id(self, max_retries=5):
         '''
-- 
2.4.3

